docs/source/user/settings.rst

```diff
@@ -603,6 +603,31 @@ via the organization radius settings section of the admin interface.
 
 .. _openwisp_radius_needs_identity_verification:
 
+``OPENWISP_RADIUS_MAC_ADDR_ROAMING_ENABLED``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Default**: ``False``
+
+Indicates whether MAC address roaming is supported.
+When this setting is enabled (i.e. ``True``),
+MAC address roaming is enabled for all organizations.
+
+**This setting can be overridden in individual organizations
+via the admin interface**, by going to *Organizations*
+then edit a specific organization and scroll down to
+*"Organization RADIUS settings"*, as shown in the screenshot below.
+
+.. image:: /images/mac-address-roaming.png
+   :alt: Organization MAC Address Roaming settings
+
+.. note::
+
+    We recommend using the override via the admin interface only when there
+    are special organizations which need a different configuration, otherwise,
+    if all the organization use the same configuration, we recommend
+    changing the global setting.
```

Comment:
is this necessary? Wouldn't this apply to all settings? Why this one in particular?

Reply:
We have this warning on other settings as well. 

https://github.com/search?q=repo%3Aopenwisp%2Fopenwisp-radius+%22We+recommend+using+the+override+via+the+admin+interface%22&type=code